docs/src/components/programs_and_authorizations.md

```diff
@@ -0,0 +1,40 @@
+# Programs and Authorizations
+
+A **Valence Program** is an instance of the **Valence Protocol**. It is a particular arrangement and configuration of [accounts](./accounts.md) and [libraries](./libraries_and_functions.md) across multiple [domains](./domains.md) e.g. a POL (protocol-owned liquidity) lending relationship between two parties. Similarly to how a _library_ exposes executable _functions_, programs are associated with a set of executable **Subroutines**.
```

Comment:
The first line is the same as in domains.md, probably worth removing it there and leaving it here?

Reply:
I think it's ok to repeat that same sentence on the two pages. It does give context for the following sentences. Also, doc pages are not always read sequentially ...